app/views/online-booking/choose-date-calendar.html

```diff
@@ -0,0 +1,383 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+  Choose date – {{ serviceName }} – GOV.UK Prototype Kit
+{% endblock %}
+
+{% block beforeContent %}
+  <a class="govuk-back-link" href="javascript:window.history.back()">Back</a>
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <form class="form" action="5reasonable-adjustments" method="post">
+
+
+        {%- from "govuk/components/tag/macro.njk" import govukTag -%}
+        {{ govukTag({
+          text: "In progress",
+          classes: "govuk-tag--purple"
+        }) }}
+        <h1 class="govuk-heading-l">
+          When do you want to visit?
+        </h1>
+
+        <p>Showing available times over the next 28 days</p>
+
+        <div id="calendar-slot-picker">
+          <h2 class="govuk-heading-m">
+            December 2023
+          </h2>
+
+          <ul class="govuk-list calendar-slots">
+            <li>M<span class="calendar-slots__day">on</span></li>
+            <li>T<span class="calendar-slots__day">ue</span></li>
+            <li>W<span class="calendar-slots__day">ed</span></li>
+            <li>T<span class="calendar-slots__day">hu</span></li>
+            <li>F<span class="calendar-slots__day">ri</span></li>
+            <li>S<span class="calendar-slots__day">at</span></li>
+            <li>S<span class="calendar-slots__day">un</span></li>
+          </ul>
+
+          <ul class="govuk-list calendar-slots">
```

Comment:
These blocks of 7 `<li>`s represent a row of dates. They can either be blank, just a number or include an `<a>` to make them clickable.

The `href` of any <a> tags needs to match the `id` of the slots lists below to tie the two together.